Subject: Allow build on Hurd, FreeBSD and GNU/kFreeBSD as well
 Actually, upstream meant to check for X11, not for Linux, but…
From: Dominik George <nik@naturalnet.de>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=852557
--- a/scan-linux.c
+++ b/scan-linux.c
@@ -1,5 +1,5 @@
 
-#ifdef linux
+#if defined (linux) || defined (__GNU__) || defined (__FreeBSD__) || defined (__FreeBSD_kernel__)
 
 #include <stdio.h>
 
